.github/actions/docker/build-api/action.yml

```diff
@@ -105,45 +80,43 @@ runs:
       id: build-image
       shell: bash
       env:
-        REGISTRY_OWNER: novuhq
-        DOCKER_NAME: ${{ inputs.docker_name }}
+        REGISTRY: ${{ steps.login-ecr.outputs.registry }}
```

Comment:
Is this action reused during OS builds?

Reply:
Do we maybe need to rename this shared YML part to make it clearer it's for the Novu Cloud?

Reply:
Sadly its only for dev 😢